src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,47 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String statisticsResult = getStatisticsFromFile(fromFileName);
+        writeTextToFile(toFileName, statisticsResult);
+    }
+
+    private String getStatisticsFromFile(String fromFileName) {
+        int buy = 0;
+        int supply = 0;
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            for (String line; (line = br.readLine()) != null; ) {
+                String[] parts = line.split(",");
+                if (parts[0].equals("buy")) {
+                    buy += Integer.parseInt(parts[1]);
+                } else {
+                    supply += Integer.parseInt(parts[1]);
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#dont-ignore-exceptions)

Reply:
add code that throws RuntimeException when we read file and write from file